samples/nodejs/nextjs-boilerplate/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.9'
+services:
+  service1:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - mode: ingress
+        target: 3000
+#secrets:
+  #API_KE
+    #external: true
```

Comment:
If this isn't in use, maybe we should delete it.

Reply:
Got it.